src/css/profile/mobile/common/datepicker.less

```diff
@@ -0,0 +1,417 @@
+.tau_datepicker_material {
+	position: absolute;
+	display: block;
+	background-color: white;
+	font-size: 14px;
+	font-family: Roboto-Regular;
+	color: #404040;
+	width: 100%;
+	ul.tau_datepicker_line {
+		position: relative;
+		display: block;
+		clear: both;
+		font: inherit;
+		margin: 0;
+		padding: 0;
+		overflow: hidden;
+		width: 100%;
+		li {
+			float: left;
+			position: relative;
+			list-style-type: none;
+			list-style-image: none;
+			text-align: center;
+			vertical-align: middle;
+			font: inherit;
+			cursor: default;
+			overflow: hidden;
+			margin: 0;
+			padding: 0;
+		}
+	}
+	div.tau_datepicker_month_cont {
+		position: relative;
+		display: block;
+		width: 100%;
+		height: 36px;
+		margin-bottom: 10px;
+		overflow: hidden;
+		font: inherit;
+		-webkit-user-select: text;
+		-khtml-user-select: text;
+		-moz-user-select: text;
+		-ms-user-select: text;
+		-o-user-select: text;
+		user-select: text;
```

Comment:
You can only use the 'user-select' because autoprefixer add the vendor prefix automatically.

Reply:
done.